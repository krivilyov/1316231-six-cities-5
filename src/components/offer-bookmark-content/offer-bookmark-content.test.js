import React from "react";
import renderer from "react-test-renderer";
import OfferBookmarkContent from "./offer-bookmark-content";

const bookmarkProperty = {
  bemBlock: `property`,
  width: `31`,
  height: `33`,
};

describe(`OfferBookmarkContent should render correctly`, () => {
  it(`OfferBookmarkContent should render correctly when offerBookmarkStatus={true}`, () => {
    const tree = renderer
      .create(
          <OfferBookmarkContent
            offerBookmarkStatus={true}
            offerBookmarkTypeProperty={bookmarkProperty}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`OfferBookmarkContent should render correctly when offerBookmarkStatus={false}`, () => {
    const tree = renderer
      .create(
          <OfferBookmarkContent
            offerBookmarkStatus={false}
            offerBookmarkTypeProperty={bookmarkProperty}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
