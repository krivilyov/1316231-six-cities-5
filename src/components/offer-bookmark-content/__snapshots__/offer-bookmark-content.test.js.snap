// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferBookmarkContent should render correctly OfferBookmarkContent should render correctly when offerBookmarkStatus={false} 1`] = `
Array [
  <svg
    className="property__bookmark-icon"
    height="33"
    width="31"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>,
  <span
    className="visually-hidden"
  >
    To
     bookmarks
  </span>,
]
`;

exports[`OfferBookmarkContent should render correctly OfferBookmarkContent should render correctly when offerBookmarkStatus={true} 1`] = `
Array [
  <svg
    className="property__bookmark-icon"
    height="33"
    width="31"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>,
  <span
    className="visually-hidden"
  >
    In
     bookmarks
  </span>,
]
`;
